src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -30,32 +30,6 @@ def _obj2txt(backend, obj):
     return backend._ffi.buffer(buf, res)[:].decode()
 
 
-def _asn1_integer_to_int(backend, asn1_int):
```

Comment:
Would you mind moving this change to a separate PR that we can review/land first? That way we can shrink the size of this one and make it a bit easier to review.